packages/sdk/src/modules/middlewareModule/utils/getRequestSender.ts

```diff
@@ -124,9 +157,24 @@ export const getRequestSender = (options: Options): RequestSender => {
       { method: finalMethod, headers, ...restConfig },
       methodConfig
     );
+    const startTime = new Date().getTime();
```

Comment:
In measuring time for responses etc. it's often used the `Performance.now()`. Do you consider using it?

Reply:
Isn't it client specific API?

Reply:
No, it's supported in NodeJS from v16 ;)
https://developer.mozilla.org/en-US/docs/Web/API/Performance

Reply:
cool! I didn't know :D updated